aws/resource_aws_ssm_maintenance_window_test.go

```diff
@@ -719,6 +753,19 @@ resource "aws_ssm_maintenance_window" "test" {
 `, rName, scheduleTimezone)
 }
 
+func testAccAWSSSMMaintenanceWindowConfigScheduleOffset(rName string, scheduleOffset int) string {
+	return fmt.Sprintf(`
+resource "aws_ssm_maintenance_window" "test" {
+  cutoff            = 1
+  duration          = 3
+  name              = %q
+  schedule          = "cron(0 16 ? * TUE *)"
+  schedule_timezone = %q
+  schedule_offset   = 3
```

Comment:
Looks like `schedule_timezone` may have been kept from the test above, which is affecting this testing in a strange way -- likely fixing this to pass through `scheduleOffset` as an integer (`%d`) will fix this:

```
=== CONT  TestAccAWSSSMMaintenanceWindow_ScheduleOffset
TestAccAWSSSMMaintenanceWindow_ScheduleOffset: resource_aws_ssm_maintenance_window_test.go:471: Step 1/3 error: Error running pre-apply refresh:
Error: Invalid character
on config467904156/terraform_plugin_test.tf line 7, in resource "aws_ssm_maintenance_window" "test":
7:   schedule_timezone = '\x02'
Single quotes are not valid. Use double quotes (") to enclose strings.
Error: Invalid expression
on config467904156/terraform_plugin_test.tf line 7, in resource "aws_ssm_maintenance_window" "test":
7:   schedule_timezone = '\x02'
Expected the start of an expression, but found an invalid expression token.
Error: Invalid character
on config467904156/terraform_plugin_test.tf line 7, in resource "aws_ssm_maintenance_window" "test":
7:   schedule_timezone = '\x02'
This character is not used within the language.
--- FAIL: TestAccAWSSSMMaintenanceWindow_ScheduleOffset (5.27s)
```

```suggestion
  cutoff          = 1
  duration        = 3
  name            = %q
  schedule        = "cron(0 16 ? * TUE *)"
  schedule_offset = %d
```